deps/chakrashim/chakracore.gyp

```diff
@@ -110,6 +110,7 @@
                 '/p:Configuration=$(ConfigurationName)',
                 '/p:RuntimeLib=<(component)',
                 '/p:AdditionalPreprocessorDefinitions=COMPILE_DISABLE_Simdjs=1',
+                '/p:WindowsTargetPlatformVersion=$(WindowsTargetPlatformVersion)',
```

Comment:
isn't this only require for ARM? See [this](https://github.com/nodejs/node-chakracore/commit/262f7e0ac649c4b1595808fd7eea688c78b6b7ee#diff-980723060c448bbe13349f168d8a0bdb)